guides/studio-development.md

```diff
@@ -0,0 +1,11 @@
+# Studio Development
+
+In production, the code used to facilitate Studio functionality will be retrieved from the Cloud. However, in order to develop locally, developers will:
+
+- Clone the `cypress-services` repo
```

Comment:
given the context of the repo this readme doesn't make a lot of sense.

Reply:
We had something similar for protocol? 🤷🏻 

https://github.com/cypress-io/cypress/blob/develop/guides/protocol-development.md

I'm fine removing those, I just thought it'd be good to have some sort of reference for how you'd develop both of them (even if it only applies to our internal team).

Reply:
As long as it is an approved existing pattern I think this is fine